src/main/resources/org/openstreetmap/atlas/utilities/command/subcommands/OsmToAtlasCommandExamplesSection.txt

```diff
@@ -0,0 +1,10 @@
+atlas osm2Atlas zz.osm
```

Comment:
hooray documentation! 👍

Reply:
yeah when I document, I put the zz at the top.

Reply:
Lmao. Took me a second to get this one.